browser/download/sources.gni

```diff
@@ -0,0 +1,18 @@
+brave_browser_download_sources = [
+  "//brave/browser/download/brave_download_item_model.cc",
+  "//brave/browser/download/brave_download_item_model.h",
+]
+
+brave_browser_download_deps = [
+  "//base",
+  "//chrome/common",
+
+  # Need for //chrome/common/safe_browsing/download_file_types.pb.h included
```

Comment:
this shouldn't be relevant anymore, we should only include the direct deps for headers in brave_download_item_model.h/.cc

Reply:
Yes, you're right and I should have noticed before proposing this PR.

Changed it now to remove that dependency on //chrome/common/safe_browsing:proto as well as the one on //chrome/common, and added a new one for //services/network/public/cpp now.

@bridiver PTAL, thanks